test/runtime/connectivity.go

```diff
@@ -93,6 +93,15 @@ var runtimeConnectivityTest = func(datapathMode string) func() {
 					return true
 				}, "Endpoints are not ready", &helpers.TimeoutConfig{Timeout: 150 * time.Second})
 				Expect(err).Should(BeNil())
+
+				err = helpers.WithTimeout(func() bool {
```

Comment:
`WaitForNPods` checks if the container is ready. Shouldn't that be enough?

Reply:
These are runtime tests, there is no k8s that provides a readiness probe